warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -314,7 +314,7 @@ public void enableYielding(YieldCallback<Key> yieldCallback) {
     
     @Override
     public void next() throws IOException {
-        ActiveQueryLog.getInstance().get(getQueryId()).beginCall(this.originalRange, ActiveQuery.CallType.NEXT);
+        ActiveQueryLog.getInstance(getActiveQueryLogName()).get(getQueryId()).beginCall(this.originalRange, ActiveQuery.CallType.NEXT);
```

Comment:
We should reduce unnecessary calls and cache accesses by declaring 
final ActiveQueryLog activeQueryLog = ActiveQueryLog.getInstance(getActiveQueryLogName());
and using  activeQueryLog in place of ActiveQueryLog.getInstance(getActiveQueryLogName()) in this method